packages/@aws-cdk/aws-appsync/README.md

```diff
@@ -334,6 +334,25 @@ new route53.CnameRecord(this, `CnameApiRecord`, {
 });
 ```
 
+## Log Group
+
+AppSync automatically create a log group with the name `/aws/appsync/apis/<graphql_api_id>` upon deployment with
+log data set to never expire. If you want to set a different expiration period, use the `logConfig.retention` property.
+
+To obtain the GraphQL API's log group as a `logs.ILogGroup` use the `logGroup` property of the
+`GraphqlApi` construct.
+
+```ts
+import * as logs from '@aws-cdk/aws-logs';
+
+new GraphqlApi(this, 'api', {
+  name: 'api',
+  logConfig: {
+    retention: logs.RetentionDays.ONE_WEEK,
+  },
+});
```

Comment:
Looks like Rosetta doesn't like the use of `api` here. Can you please update to something else? Also, It looks like some of the fields are missing. The code snippet you put in the integration test would work great here, if you want to just copy/paste that.